### DISCUSSION TITLE
how to install markdown-preview.nvim ?

### DISCUSSION THREAD

**Discussion by lmezg** [NvChad/NvChad#1827] [General]
https://github.com/iamcco/markdown-preview.nvim
with the latest 2.0 branch, i put these:
```
{
  "iamcco/markdown-preview.nvim",
  ft = {"markdown"},
  run = "cd app && yarn install",
},
```
in custom/plugins.lua, the plugin was installed successfully, but it's not working because I think run = "cd app && yarn install" was not executed. I can not find any documents about how to run this command during setup. As a reference, here're the descriptions when using packer:
```
-- install without yarn or npm
use({
    "iamcco/markdown-preview.nvim",
    run = function() vim.fn["mkdp#util#install"]() end,
})

use({ "iamcco/markdown-preview.nvim", run = "cd app && npm install", setup = function() vim.g.mkdp_filetypes = { "markdown" } end, ft = { "markdown" }, })
```

**Comment by siduck** [comment]
hi @gengjun  use build instead of run https://github.com/folke/lazy.nvim#packernvim

**Reply by lmezg** [reply]
change run to build works! Thanks!
